SPECS/attr/attr.spec

```diff
@@ -1,53 +1,46 @@
-Summary:       Utilities for managing filesystem extended attributes
-Name:          attr
-Vendor:        Microsoft Corporation
-Group:         System Environment/Security
-Distribution:  Mariner
-Version:       2.4.48
-Release:       6%{?dist}
-Source:        https://download-mirror.savannah.gnu.org/releases/attr/attr-%{version}.tar.gz
-%define sha1 %{name}=d7c88cb5db51c5e255a511b0a15e832fcdda5b4f
-
-# fix test-suite failure with perl-5.26.0 (#1473853)
-Patch1:  0001-attr-2.4.48-test-suite-perl.patch
-
-# fix conflict with fakechroot (https://github.com/dex4er/fakechroot/issues/57)
-Patch2:  0002-attr-2.4.48-switch-back-to-syscall.patch
-
-License: GPLv2+
-URL: https://savannah.nongnu.org/projects/attr
-BuildRequires: gettext
-BuildRequires: libtool
-BuildRequires: flex-devel
-Requires: libattr = %{version}-%{release}
+Summary:        Utilities for managing filesystem extended attributes
+Name:           attr
+Version:        2.5.1
+Release:        1%{?dist}
+License:        GPLv2+ AND LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://savannah.nongnu.org/projects/attr
+Source:         http://download.savannah.nongnu.org/releases/%{name}/%{name}-%{version}.tar.gz
```

Comment:
Nit: seems like this could be HTTPS (applies to cgmanifest, as well)

Reply:
Thanks, I missed that one.